react/components/EXPERIMENTAL_FilterBar/UncoupledStatement.js

```diff
@@ -0,0 +1,144 @@
+              isFullWidth ? 'flex-column items-stretch' : ''
+            }`}>
+            {omitSubject
+              ? omitVerbs
+                ? [<ObjectAtom key="object" {...atomProps} />]
+                : [
+                    condition.subject && (
+                      <VerbAtom
+                        key="verb"
+                        {...atomProps}
+                        onChangeStatement={(value, structure) => {
+                          this.handleChangeStatement(value, structure)
+                        }}
+                      />
+                    ),
+                    condition.verb && (
+                      <ObjectAtom key="object" {...atomProps} />
+                    ),
+                  ]
+              : [
+                  <SubjectAtom
+                    key="subject"
+                    {...atomProps}
+                    placeholder={subjectPlaceholder}
+                    onChangeStatement={(value, structure) => {
+                      this.handleChangeStatement(value, structure)
+                      this.resetPredicate(value)
+                    }}
+                  />,
+                  condition.subject && (
+                    <VerbAtom
+                      key="verb"
+                      {...atomProps}
+                      onChangeStatement={(value, structure) => {
+                        this.handleChangeStatement(value, structure)
+                      }}
+                    />
+                  ),
+                  condition.verb && <ObjectAtom key="object" {...atomProps} />,
+                ]}
+          </div>
+          {condition.error && condition.error.message && (
+            <div className="red t-small mh3 mt2 lh-title">
+              {condition.error.message}
+            </div>
+          )}
+        </div>
+      </div>
+    )
+  }
+}
+
+Statement.defaultProps = {
+  onRemoveStatement: () => {},
+  onChangeStatement: () => {},
+  onChangeObjectCallback: () => {},
+  statements: [{ subject: '', verb: '', object: null, error: null }],
+  isFullWidth: false,
+  statementIndex: 0,
+  omitSubject: false,
+  omitVerbs: false,
+}
+
+Statement.propTypes = {
+  /** Current selected options for this Statement */
+  statements: PropTypes.arrayOf(
+    PropTypes.shape({
+      subject: PropTypes.string,
+      verb: PropTypes.string,
+      object: PropTypes.any,
+      error: PropTypes.string,
+    })
+  ),
+  /** Possible options and respective data types, verb options */
+  options: PropTypes.object.isRequired,
+  /** Placeholder for subject dropdown */
+  subjectPlaceholder: PropTypes.string.isRequired,
+  /** Stretch component to 100% of the width */
+  isFullWidth: PropTypes.bool,
+  /** Statement change callback */
+  onChangeStatement: PropTypes.func,
+  /** To which row does this Statement belong to?  */
+  statementIndex: PropTypes.number,
+  /** Please use the 3 following ones with caution, i did not test them, so they can break everything */
```

Comment:
Is it hard to test these?

Reply:
yes 😞. I would have to develop this feature in the Conditions component to see if the prop fits or break it. Instead I made another component, so I guess there is no danger anymore. I'll remove the comment